src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,72 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
 
     public void getStatistic(String fromFileName, String toFileName) {
+        writeToFile(fromFileName, toFileName);
+    }
+
+    private String readFromFile(String fromFileName) {
+        StringBuilder stringBuilder = new StringBuilder();
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(fromFileName))) {
+            String dataFromFile = bufferedReader.readLine();
+            while (dataFromFile != null) {
+                stringBuilder.append(dataFromFile).append(" ");
+                dataFromFile = bufferedReader.readLine();
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("File doesn't exist", e);
+        }
+        return stringBuilder.toString();
+    }
+
+    private void writeToFile(String fromFileName, String toFileName) {
```

Comment:
This method must just write a complete report, don't create a report inside
Same with all methods